content/en/real_user_monitoring/mobile_and_tv_monitoring/android/setup.md

```diff
@@ -417,8 +417,31 @@ OkHttpClient okHttpClient = new OkHttpClient.Builder()
 {{% /tab %}}
 {{< /tabs >}}
 
+If you want RUM resources and Spans to be automatically created for your OkHttp requests, you can use the `DatadogInterceptor` as a network interceptor.
+
 This records each request processed by the `OkHttpClient` as a resource, with all the relevant information (URL, method, status code, and error) automatically filled in. Only the network requests that started when a view is active are tracked. To track requests when your application is in the background, [create a view manually][13].
 
+In case you want to monitor the network redirects or retries, you can use the `DatadogInterceptor` as a network interceptor.
```

Comment:
maybe it is worth to link OkHttp documentation on the difference between `interceptor` and `networkInterceptor`?

Reply:
not sure, I am waiting for suggestions here from doc reviewers.

Reply:
We link to the OkHttp docs early in step 3, right before the tabs that you can see the closing tags for at the very top of this PR. I think that's close enough to this note to be useful, but I'm not against adding the link again if you're concerned about users getting the two confused.

Reply:
Yes, linking to the OkHttp docs is a good idea. I would put the link on the phrase "network interceptor."